argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml

```diff
@@ -10,17 +10,22 @@ spec:
         generators:
           - clusters:
               values:
-                environment: ""
-                clusterDir: ""
+                sourceRoot: configs/ingresscontroller
+                environment: staging
+                clusterDir: base
```

Comment:
By defaulting all `clusters to configs/ingresscontroller/base`, this means that the time out patch will be applied to all clusters, not only dev, stg-host and prd-host1. Is this what you want?

Reply:
ignore my comment, we detault to configs/ingresscontroller/staging/base which does not exists. I think we are good